sdk/monitor/opentelemetry-exporter/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@azure/monitor-opentelemetry-exporter",
+  "author": "appinsightssdk@microsoft.com",
+  "version": "1.0.0-preview.4",
+  "description": "Application Insights exporter for the OpenTelemetry JavaScript (Node.js) SDK",
+  "main": "dist-esm/src/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "types/src/index.d.ts",
+  "scripts": {
```

Comment:
rush might break if you don't have at least stub versions of the common build/test tasks we use, since it blindly applies them to all projects without detecting if they exist

Reply:
I've added some stubs here, but will solve in separate issue once rush integration is done